extensions/wikia/Wall/Wall.setup.php

```diff
@@ -255,4 +262,13 @@
 $wgGroupPermissions['staff']['wallfastadmindelete'] = true;
 
 $wgGroupPermissions['*']['wallmessagemove'] = false;
-$wgGroupPermissions['user']['wallmessagemove'] = true;
+$wgGroupPermissions['user']['wallmessagemove'] = false;
+$wgGroupPermissions['moderator']['wallmessagemove'] = true;
+$wgGroupPermissions['sysop']['wallmessagemove'] = true;
+$wgGroupPermissions['vstf']['wallmessagemove'] = true;
+$wgGroupPermissions['helper']['wallmessagemove'] = true;
+$wgGroupPermissions['staff']['wallmessagemove'] = true;
+
+// VOLDEV-79
```

Comment:
Adding this comment again for this pull request:

This comment is a little cryptic and looks a little like commented out code rather than a code example of what was added. Given that none of the other permissions have explanations or tickets associated with them and we can use GitHub's tools to see where/why lines were added, we can probably remove this.